docs/product-ibm-websphere-installation-and-configuration-guide/src/main/asciidoc/controller-was-install-proc.adoc

```diff
@@ -3,8 +3,14 @@
 
 To use the {KIE_SERVER} REST API or Java Client API to interact with {KIE_SERVER}, you can install the {KIE_SERVER} Controller with {WEBSPHERE}. The {KIE_SERVER} Controller manages {KIE_SERVER} configuration in a centralized way so that you can use the Controller to create and maintain containers and perform other server-level tasks.
 
+[NOTE]
+====
+Red Hat recommends that you install {KIE_SERVER} and the {KIE_SERVER} Controller on different servers in production environments. In development environments, you can install {KIE_SERVER} and the {KIE_SERVER} Controller on the same server.
+====
+
 .Prerequisites
-* An {WEBSPHERE} instance is configured as described in xref:was-configure-proc[].
+* The {WEBSPHERE} instance is configured as described in xref:was-configure-proc[].
+* {KIE_SERVER} is installed on the {WEBSPHERE} instance.
```

Comment:
Should this be a prerequisite? AFAIK the documentation should describe the deployment where standalone controller is installed on different WAS than Decision Server.